src/components/policy/policy_external/src/sql_pt_queries.cc

```diff
@@ -678,9 +681,10 @@ const std::string kInsertApplication =
     "INSERT OR IGNORE INTO `application` (`id`, `priority_value`, "
     "`is_revoked`, `memory_kb`, `heart_beat_timeout_ms`, `certificate`, "
     "`hybrid_app_preference_value`, `endpoint`, `enabled`, `auth_token`, "
-    "`cloud_transport_type`, `icon_url`, `allow_unknown_rpc_passthrough`) "
+    "`cloud_transport_type`, `icon_url`, `allow_unknown_rpc_passthrough`, "
+    "`encryption_required`) "
```

Comment:
There are also some similar changes that need to be added to the sql_pt_ext_queries.cc. See kSelectAppPolicies and kInsertApplication

Reply:
Added in 988d4fca0c6a9825e3e2c301905c00705bbcc901